_inc/lib/tracks/class.tracks-client.php

```diff
@@ -184,6 +184,9 @@ static function get_connected_user_tracks_identity() {
 		}
 
 		return array(
+			'blogid' => ! empty( Jetpack_Options::get_option( 'id' ) )
+				? Jetpack_Options::get_option( 'id' )
```

Comment:
This is causing CI failures on PHP 5.2. 5baea012feed2f792ab395d7ba5c5b183ae0e68d contains a fix but it looks like that was lost.

Reply:
Reapplying.